contract/src/offer-up.contract.js

```diff
@@ -72,6 +73,16 @@ export const customTermsShape = meta.customTermsShape;
 export const start = async zcf => {
   const { tradePrice, maxItems = 3n } = zcf.getTerms();
 
+  /**
+   * TODO: below is somewhat fake way of generating a zone. This should be fixed by passing it as a parameter.
+   * RN I don't know how to do it.
+   */
+  const baggage = makeScalarBigMapStore('baggage', {
+    keyShape: M.string(),
+    durable: true,
+  });
+  const zone = makeDurableZone(baggage);
+
```

Comment:
Baggage is the 3rd arg of start:

```suggestion
export const start = async (zcf, _privateArgs, baggage) => {
  const zone = makeDurableZone(baggage);

```

this is (supposed to be) documented in the [Upgradable Declaration](https://docs.agoric.com/guides/zoe/contract-upgrade.html#upgradable-declaration) of [Contract Upgrade](https://docs.agoric.com/guides/zoe/contract-upgrade.html).  That section documents a older pattern using a `prepare` function. It still works, but it's not what we want to teach. See 
 - https://github.com/Agoric/documentation/issues/1126